### ISSUE TITLE
404 page is coming up after SSO login

### ISSUE THREAD

**Issue by vikassinha2019** [Netflix/dispatch#1443]
We are trying to up SSO login for Dispatch using OpenID. Following are the config we have done for that:

./src/auth/pkceAuthProvider.js
```
 20   const clientId = "15xxxxxx-2xxx-4xxx-bxxx-3xxxxxxxx"
 21   const openIdConnectUrl =
 22     "https://login.microsoftonline.com/dxxxxxx-bxxx-4xxx-8xxx-902cxxxxxx/v2.0"
```
```
 78             store.dispatch("auth/login", {
 79              token: response.idToken
```
  
./src/dispatch/static/dispatch/.env
```
# Authentication
VUE_APP_DISPATCH_OPEN_ID_CONNECT_URL="https://login.microsoftonline.com/dxxxxxx-bxxx-4xxx-8xxx-902cxxxxxx/oauth2/v2.0/authorize"
VUE_APP_DISPATCH_CLIENT_ID="15xxxxxx-2xxx-4xxx-bxxx-3xxxxxxxx"
VUE_APP_DISPATCH_AUTHENTICATION_PROVIDER_SLUG="dispatch-auth-provider-pkce"
```

And the respective configs in dispatch-docker/.env file.

We are able to get to SSO login page, and traverse through enter credentials pages. However, after the authentication it is responding with following page. None of the network trace call is showing failed with 404 though. 

<img width="950" alt="Screenshot 2021-07-15 at 7 51 49 PM" src="https://user-images.githubusercontent.com/52274121/125804227-1298158f-bf4d-42e6-855e-4fa9733d1c3a.png">

Require help getting past this issue. TIA
cc: @kevgliss

**Comment by kevgliss** [comment]
What version of dispatch are you using? Do you have any errors in the console?

That path does exist in the app here:
https://github.com/Netflix/dispatch/blob/master/src/dispatch/static/dispatch/src/router/config.js#L46

You wouldn't see network call as the frontend handles the route client-side.

**Comment by vikassinha2019** [comment]
With the latest version, I am not getting the SSO page itself. Hence, using the second latest version v20210603. 
No errors are showing up on the console. Attaching log snippet for reference. 
[out.txt](https://github.com/Netflix/dispatch/files/6836841/out.txt)

**Comment by kevgliss** [comment]
Do you have `DISPATCH_AUTHENTICATION_PROVIDER_SLUG=dispatch-auth-provider-pkce` set in your config?

fwiw I just tried to re-create your issue on master an wasn't able to (although we have our own pkce provider not Microsoft).

**Comment by vikassinha2019** [comment]
Yes, the DISPATCH_AUTHENTICATION_PROVIDER_SLUG=dispatch-auth-provider-pkce config was added in .env file.
